src/cpp/r/RSexp.cpp

```diff
@@ -172,24 +196,58 @@ SEXP findVar(const std::string& name, const std::string& ns)
    
    return findVar(name, env);
 }
-  
+
 SEXP findFunction(const std::string& name, const std::string& ns) 
 {
+   r::sexp::Protect protect;
```

Comment:
My intention here -- I wanted to be able to call `findFunction` on a symbol name that might not exist, without getting an R error thrown when that happened (otherwise that error gets printed to the console). With this change, since `R_UnboundValue` is returned on failure, I can handle that locally (and avoid printing anything to the console).